src/index.css

```diff
@@ -131,6 +131,10 @@ code {
   padding-right: 10px !important;
 }
 
+.p-r-20 {
+  padding-right: 20px !important;
+}
+
```

Comment:
`margin-right`, also don't put arbitrary gaps in index.css, instead, create `module.css` file and put it there.

Reply:
both options are possible, for simple cases when module.css is not needed generics seems to be reasonable too